python/deltalake/writer.py

```diff
@@ -147,13 +147,16 @@ def write_deltalake(
         else:
             schema = data.schema
 
+    if filesystem is not None:
+        raise NotImplementedError("Filesystem support is not yet implemented.  #570")
```

Comment:
I made a note of what we have to do here: https://github.com/delta-io/delta-rs/issues/570#issuecomment-1288249167